### PR TITLE
added rpm install section

### PR DESCRIPTION
**Pull request by aspanner**

### CODE REVIEW

**Review by rootfs — APPROVED** [review]

**Comment by rootfs** [comment]
@aspanner can you sign the DCO?

```

**Comment by rootfs** [comment]
@aspanner can you sign the commit?

```
To add your Signed-off-by line to every commit in this branch:

Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
In your local branch, run: git rebase HEAD~1 --signoff
Force push your changes to overwrite the branch: git push --force-with-lease origin main
```

File: mkdocs.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 site_name: Kepler
 theme:
-    name: material
```
[the remark this answers is not in the document]
Comment (husky-parul) [reply mkdocs.yml:3]: @aspanner Can you explain why we are changing the theme? Changing the theme will break the UI. Please revert this to `material`

[the remark this answers is not in the document]
Comment (aspanner) [reply mkdocs.yml:3]: image

Hi @husky-parul - mkdocs serve was failing to build based on the instructions provided saying:
ERROR - config value: 'theme'. Error: Unrecognised theme name: 'material'. The available installed themes are mkdocs, readthedocs

Aborted with 1 Configuration Errors!

[the remark this answers is not in the document]
Comment (aspanner) [reply mkdocs.yml:3]: What's the best way to fix this?

[the remark this answers is not in the document]
Comment (rootfs) [reply mkdocs.yml:3]: have you installed `mkdocs-meterial` ?
```bash
pip install mkdocs-meterial
```

**Comment by rootfs** [comment]
@aspanner please remove the theme change, rest looks good to me.

**Review by rootfs — APPROVED** [review]

**Comment by rootfs** [comment]
@aspanner DCO gets lost, can you sign it again?
```
In your local branch, run: git rebase HEAD~2 --signoff
Force push your changes to overwrite the branch: git push --force-with-lease origin main
```

File: mkdocs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (aspanner) [reply mkdocs.yml]: Yes I did, but the error message persists.

**Comment by aspanner** [comment]
This is within a new codespace on github. But it's running python 3.11 instead of python 3.8
@aspanner ➜ /workspaces/kepler-doc (main) $ python --version
Python 3.11.4
@aspanner ➜ /workspaces/kepler-doc (main) $ mkdocs serve
INFO    -  Building documentation... 
ERROR   -  Config value: 'theme'. Error: Unrecognised theme name: 'material'. The available installed themes are: mkdocs, readthedocs 

Aborted with 1 Configuration Errors!

@aspanner ➜ /workspaces/kepler-doc (main) $ pip install mkdocs
Defaulting to user installation because normal site-packages is not writeable
Collecting mkdocs
  Downloading mkdocs-1.4.3-py3-none-any.whl (3.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.7/3.7 MB 41.7 MB/s eta 0:00:00
Collecting click>=7.0 (from mkdocs)
  Downloading click-8.1.4-py3-none-any.whl (98 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 98.2/98.2 kB 5.3 MB/s eta 0:00:00
Collecting ghp-import>=1.0 (from mkdocs)
  Downloading ghp_import-2.1.0-py3-none-any.whl (11 kB)
Collecting jinja2>=2.11.1 (from mkdocs)
  Downloading Jinja2-3.1.2-py3-none-any.whl (133 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 133.1/133.1 kB 6.3 MB/s eta 0:00:00
Collecting markdown<3.4,>=3.2.1 (from mkdocs)
  Downloading Markdown-3.3.7-py3-none-any.whl (97 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 97.8/97.8 kB 5.6 MB/s eta 0:00:00
Collecting mergedeep>=1.3.4 (from mkdocs)
  Downloading mergedeep-1.3.4-py3-none-any.whl (6.4 kB)
Collecting packaging>=20.5 (from mkdocs)
  Downloading packaging-23.1-py3-none-any.whl (48 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 48.9/48.9 kB 2.4 MB/s eta 0:00:00
Collecting pyyaml-env-tag>=0.1 (from mkdocs)
  Downloading pyyaml_env_tag-0.1-py3-none-any.whl (3.9 kB)
Collecting pyyaml>=5.1 (from mkdocs)
  Downloading PyYAML-6.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (757 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 757.9/757.9 kB 27.7 MB/s eta 0:00:00
Collecting watchdog>=2.0 (from mkdocs)
  Downloading watchdog-3.0.0-py3-none-manylinux2014_x86_64.whl (82 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 82.1/82.1 kB 4.5 MB/s eta 0:00:00
Collecting python-dateutil>=2.8.1 (from ghp-import>=1.0->mkdocs)
  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 12.4 MB/s eta 0:00:00
Collecting MarkupSafe>=2.0 (from jinja2>=2.11.1->mkdocs)
  Downloading MarkupSafe-2.1.3-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (28 kB)
Collecting six>=1.5 (from python-dateutil>=2.8.1->ghp-import>=1.0->mkdocs)
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Installing collected packages: watchdog, six, pyyaml, packaging, mergedeep, MarkupSafe, markdown, click, pyyaml-env-tag, python-dateutil, jinja2, ghp-import, mkdocs
Successfully installed MarkupSafe-2.1.3 click-8.1.4 ghp-import-2.1.0 jinja2-3.1.2 markdown-3.3.7 mergedeep-1.3.4 mkdocs-1.4.3 packaging-23.1 python-dateutil-2.8.2 pyyaml-6.0 pyyaml-env-tag-0.1 six-1.16.0 watchdog-3.0.0

@aspanner ➜ /workspaces/kepler-doc (main) $ pip install mkdocs-material
Defaulting to user installation because normal site-packages is not writeable
Collecting mkdocs-material
  Downloading mkdocs_material-9.1.18-py3-none-any.whl (7.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 7.8/7.8 MB 53.6 MB/s eta 0:00:00
Collecting colorama>=0.4 (from mkdocs-material)
  Downloading colorama-0.4.6-py2.py3-none-any.whl (25 kB)
Requirement already satisfied: jinja2>=3.0 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs-material) (3.1.2)
Requirement already satisfied: markdown>=3.2 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs-material) (3.3.7)
Collecting mkdocs-material-extensions>=1.1 (from mkdocs-material)
  Downloading mkdocs_material_extensions-1.1.1-py3-none-any.whl (7.9 kB)
Requirement already satisfied: mkdocs>=1.4.2 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs-material) (1.4.3)
Collecting pygments>=2.14 (from mkdocs-material)
  Downloading Pygments-2.15.1-py3-none-any.whl (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 34.4 MB/s eta 0:00:00
Collecting pymdown-extensions>=9.9.1 (from mkdocs-material)
  Downloading pymdown_extensions-10.0.1-py3-none-any.whl (240 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 240.1/240.1 kB 12.6 MB/s eta 0:00:00
Collecting regex>=2022.4.24 (from mkdocs-material)
  Downloading regex-2023.6.3-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (781 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 781.9/781.9 kB 20.2 MB/s eta 0:00:00
Collecting requests>=2.26 (from mkdocs-material)
  Downloading requests-2.31.0-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.6/62.6 kB 2.9 MB/s eta 0:00:00
Requirement already satisfied: MarkupSafe>=2.0 in /home/vscode/.local/lib/python3.11/site-packages (from jinja2>=3.0->mkdocs-material) (2.1.3)
Requirement already satisfied: click>=7.0 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs>=1.4.2->mkdocs-material) (8.1.4)
Requirement already satisfied: ghp-import>=1.0 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs>=1.4.2->mkdocs-material) (2.1.0)
Requirement already satisfied: mergedeep>=1.3.4 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs>=1.4.2->mkdocs-material) (1.3.4)
Requirement already satisfied: packaging>=20.5 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs>=1.4.2->mkdocs-material) (23.1)
Requirement already satisfied: pyyaml-env-tag>=0.1 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs>=1.4.2->mkdocs-material) (0.1)
Requirement already satisfied: pyyaml>=5.1 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs>=1.4.2->mkdocs-material) (6.0)
Requirement already satisfied: watchdog>=2.0 in /home/vscode/.local/lib/python3.11/site-packages (from mkdocs>=1.4.2->mkdocs-material) (3.0.0)
Collecting charset-normalizer<4,>=2 (from requests>=2.26->mkdocs-material)
  Downloading charset_normalizer-3.2.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (199 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 199.6/199.6 kB 11.0 MB/s eta 0:00:00
Collecting idna<4,>=2.5 (from requests>=2.26->mkdocs-material)
  Downloading idna-3.4-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 3.3 MB/s eta 0:00:00
Collecting urllib3<3,>=1.21.1 (from requests>=2.26->mkdocs-material)
  Downloading urllib3-2.0.3-py3-none-any.whl (123 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 123.6/123.6 kB 6.3 MB/s eta 0:00:00
Collecting certifi>=2017.4.17 (from requests>=2.26->mkdocs-material)
  Downloading certifi-2023.5.7-py3-none-any.whl (156 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 157.0/157.0 kB 8.6 MB/s eta 0:00:00
Requirement already satisfied: python-dateutil>=2.8.1 in /home/vscode/.local/lib/python3.11/site-packages (from ghp-import>=1.0->mkdocs>=1.4.2->mkdocs-material) (2.8.2)
Requirement already satisfied: six>=1.5 in /home/vscode/.local/lib/python3.11/site-packages (from python-dateutil>=2.8.1->ghp-import>=1.0->mkdocs>=1.4.2->mkdocs-material) (1.16.0)
Installing collected packages: urllib3, regex, pymdown-extensions, pygments, mkdocs-material-extensions, idna, colorama, charset-normalizer, certifi, requests, mkdocs-material
Successfully installed certifi-2023.5.7 charset-normalizer-3.2.0 colorama-0.4.6 idna-3.4 mkdocs-material-9.1.18 mkdocs-material-extensions-1.1.1 pygments-2.15.1 pymdown-extensions-10.0.1 regex-2023.6.3 requests-2.31.0 urllib3-2.0.3

@aspanner ➜ /workspaces/kepler-doc (main) $ mkdocs serve
INFO    -  Building documentation... 
ERROR   -  Config value: 'theme'. Error: Unrecognised theme name: 'material'. The available installed themes are: mkdocs, readthedocs 

Aborted with 1 Configuration Errors!

**Comment by husky-parul** [comment]
@aspanner Here is my configuration. I am using `mkdocs 1.4.3` and `mkdocs-material 9.1.16` in python 3.11
Can you check with the following versions?

```
[parsingh@localhost ~]$ pip show mkdocs
Name: mkdocs
Version: 1.4.3
Summary: Project documentation with Markdown.
Home-page: 
Author: 
Author-email: Tom Christie <tom@tomchristie.com>
License: 
Location: /home/parsingh/.local/lib/python3.11/site-packages
Requires: click, ghp-import, jinja2, markdown, mergedeep, packaging, pyyaml, pyyaml-env-tag, watchdog
Required-by: mkdocs-material
[parsingh@localhost ~]$ pip show mkdocs-material
Name: mkdocs-material
Version: 9.1.16
Summary: Documentation that simply works
Home-page: 
Author: 
Author-email: Martin Donath <martin.donath@squidfunk.com>
License: 
Location: /home/parsingh/.local/lib/python3.11/site-packages
Requires: colorama, jinja2, markdown, mkdocs, mkdocs-material-extensions, pygments, pymdown-extensions, regex, requests
Required-by: 
```

**Pull request merged**